psi4/src/read_options.cc

```diff
@@ -302,6 +304,80 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_str("PCM_CC_TYPE", "PTE", "PTE");
     }
 
+    if (name == "DDX" || options.read_globals()) {
+        /*- MODULEDESCRIPTION Performs continuum solvation model computations using
+            the domain-decomposition paradigm. -*/
+
+        /*- Switch available solvation models -*/
+        options.add_str("MODEL", "PCM", "PCM COSMO");
+
+        /*- Radius set for cavity spheres. Ignored if RADII is set. -*/
+        options.add_str("RADII_SET", "UFF", "UFF BONDI");
+
+        /*- Scaling factor for cavity spheres. Ignored if RADII is set. The default depends on
+            the radii set chosen. -*/
+        options.add_double("RADII_SCALING", 1.1);
+
+        /*- Custom cavity radii. One per atom, uses the unit of the molecule. -*/
+        options.add("RADII", new ArrayType());
+
+        /*- Solvent to use. Not case sensitive. Ignored if SOLVENT_EPSILON is set. -*/
+        options.add_str("SOLVENT", "");
+
+        /*- Dielectric constant of the solvent to use -*/
+        options.add_double("SOLVENT_EPSILON", 0);
+
+        /*- Maximal degree of modelling spherical harmonics -*/
+        options.add_int("LMAX", 9);
+
+        /*- Number of Lebedev grid points to use.
+            (A :ref:`Lebedev Points <table:lebedevorder>` number) -*/
+        options.add_int("N_LEBEDEV", 302);
+
+        /*- Maximal number of iterations used inside DDX -*/
+        options.add_int("MAXITER", 100);
+
+        /*- Number of previous iterates to use in DIIS acceleration inside DDX -*/
+        options.add_int("DIIS_MAX_VECS", 20);
+
+        /*- Tolerance to which DDX linear systems are solved -*/
+        options.add_double("SOLVATION_CONVERGENCE", 1e-8);
+
+        /*- Number of spherical points used to compute the integrals for DDX calculations
+            (A :ref:`Lebedev Points <table:lebedevorder>` number) -*/
+        options.add_int("DFT_SPHERICAL_POINTS", 110);
+
+        /*- Number of radial points used to compute the integrals for DDX calculations -*/
+        options.add_int("DFT_RADIAL_POINTS", 35);
```

Comment:
`ddx__dft_radial_points` is the logical option name the way the options class was written. But, a flatter options namespace has become preferred over the years, so feel free to do `ddx__ddx_radial_points` like https://github.com/psi4/psi4/blob/master/psi4/src/read_options.cc#L280-L283 so that users don't necessarily have to set the module.

Reply:
Ok. I've put it into global.